package-lock.json

```diff
@@ -32,6 +32,7 @@
       "dependencies": {
         "@mdi/js": "^5.9.55",
         "kv-tokens": "^0.0.1",
+        "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.1.0",
```

Comment:
Is there a change to package.json that needs to be committed?

Reply:
I've don't have any package.json changes, this was added when you and I were on zoom earlier today. Without this line lint throws this error and prevents me from committing my code: 

/Users/joshua/kiva/kv-ui-elements/kv-tokens/configs/tailwind.config.js
  1:24  error  'tailwindcss' should be listed in the project's dependencies. Run 'npm i -S tailwindcss' to add it  import/no-extraneous-dependencies

✖ 1 problem (1 error, 0 warnings)